src/lib.rs

```diff
@@ -182,13 +184,19 @@ impl Navigator {
             .expect("Error : Error on magnetometer during self-test")
     }
 
-    pub fn set_pwm_value(&mut self) {
-        let channel = pwm_Channel::C0; //todo import enums
-        let value = 2047; //todo a fomula that validades from 0 to 100.00% and convert to 4092
-        self.pwm.set_channel_on(channel, 0).unwrap(); //Channel::C0
+    pub fn set_pwm_value(&mut self, channel: pwm_Channel, value: u16) {
+        self.pwm.set_channel_on(channel, 0).unwrap();
         self.pwm.set_channel_off(channel, value).unwrap();
     }
 
+    pub fn set_pwm_value_mult(&mut self, channels: Option<&[pwm_Channel]>, value: u16) {
```

Comment:
Why is this an `Option` ?

Reply:
removed it, was thinking to skip if no [] provided, or maybe use all_channel